src/api/app/views/webui2/webui/project/_pulse_list_comments.html.haml

```diff
@@ -0,0 +1,10 @@
+- if @comments.any?
+  - people = @comments.group(:user_name).count
+  %b
+    = pluralize(people.count, 'person')
```

Comment:
shouldn't this be just ''people''? without the `count`

Reply:
No